third-party/rspamd/docker-compose-distributed.yml

```diff
@@ -108,7 +108,7 @@ services:
       redis:
         condition: service_started
     container_name: rspamd
-    image: a16bitsysop/rspamd:3.8.4-r0-alpine3.20.0-r0
+    image: rspamd/rspamd:3.9.1
     environment:
       - REDIS=redis
```

Comment:
Do we need to update the related environment?

Reply:
Yes. Good catch.

Reply:
RspamdHealthcheckTest actually failing yes, need to adapt

Reply:
> RspamdHealthcheckTest actually failing yes, need to adapt

fixed